packages/hub/package.json

```diff
@@ -71,6 +71,7 @@
   },
   "scripts": {
     "start": "node ./bin/server.ts",
+    "console": "node --experimental-repl-await ./bin/console.ts",
```

Comment:
`--experimental-repl-await` flag allows await to be used "freely", without the async function